### PR TITLE
Update README notes on email sending, and environment variable configuration

### PR DESCRIPTION
**Pull request by anaulin**
An update to our README, motivated by our recent Quest to Find Sent Emails 🛡️⚔️.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -126,34 +126,43 @@ If you prefer to do deploys from the command line, here are the steps:
...
+
+# For Google OAuth
 GOOGLE_CLIENT_ID
 GOOGLE_CLIENT_SECRET
-HEROKU_POSTGRESQL_CYAN_URL
+
+# Same as DATABASE_URL
 HEROKU_POSTGRESQL_RED_URL
+
+# To receive due payments via Stripe
```
Comment (brittag) [inline README.md:154]: due -> dues

File: README.md [hunk 2]
```diff
@@ -164,8 +173,7 @@ TODO: It would be great to document these variables further, and figure out whic
 
 ### Email
 
-This app sends emails via AWS: TODO more info here (we also have env variables set for Mandrill, are these still needed?)
-
+This app sends emails via the Amazon SES service. Look for the `config.action_mailer.delivery_method` in [production.rb](config/environments/production.rb) for the precise configuration details.
```
Comment (brittag) [inline README.md:176]: Might be helpful to add: "If you need more information about our AWS setup, board members have console access."

**Comment by brittag** [comment]
This is great, thank you Ana! Glad to have helped clear up some of these questions.

File: README.md (hunk 2 above)
Comment (anaulin) [reply README.md:176]: Done.

File: README.md (hunk 1 above)
Comment (anaulin) [reply README.md]: Fixed.

**Comment by sarahwalters** [comment]
Thank you Ana!! I understand what's going on with the emails now.

**Pull request merged**